fplus-http-server/src/router/application.rs

```diff
@@ -23,7 +23,10 @@ pub async fn create(info: web::Json<CreateApplicationInfo>) -> impl Responder {
 pub async fn single(query: web::Query<ApplicationQueryParams>) -> impl Responder {
     let ApplicationQueryParams { id, owner, repo } = query.into_inner();
     match LDNApplication::load_from_db(id, owner, repo).await {
-        Ok(app_file) => HttpResponse::Ok().body(serde_json::to_string_pretty(&app_file).unwrap()),
+        Ok(app_file) => match serde_json::to_string_pretty(&app_file) {
+            Ok(response) => HttpResponse::Ok().body(response),
+            Err(_) => HttpResponse::InternalServerError().body("Failed to parse success message"),
```

Comment:
```suggestion
            Err(_) => HttpResponse::InternalServerError().body("Failed to serialize success message"),
```

Reply:
Applies to all cases where it says `parse` instead of `serialize` when refering to `to_string_pretty`.